docs/using-radicle/creating-patches.md

```diff
@@ -38,7 +38,7 @@ project with the **Checkout** option on your project page.
 
 Once you've checked out your fork of the project you'd like to contribute to,
 you can **create a patch**. A patch is a set of proposed changes to the source
-code of a project. Patches can be accepted & merged or rejected by a project's
+code of a project. Patches can be accepted & merged or rejected by project's
 maintainers. Similar to pull requests and merge requests, patches are used to
```

Comment:
I think the correct fix here would be to remove the plural s on “maintainers” instead of dropping the “a”.

Reply:
✅